tests/testthat/test-content.R

```diff
@@ -168,6 +168,16 @@ without_internet({
       con$content(name = "A name for content"),
       "https://connect.example/__api__/v1/content?name=A%20name%20for%20content&include=tags%2Cowner"
     )
+
+    expect_GET(
+      con$content(guid = 1),
+      glue::glue("https://connect.example/__api__/v1/content/1?include=tags%2Cowner")
+    )
+
+    expect_GET(
+      con$content(guid = 1, include = "owner,tags"),
+      glue::glue("https://connect.example/__api__/v1/content/1?include=owner%2Ctags")
```

Comment:
We don't need to glue anymore, right?

```suggestion
      "https://connect.example/__api__/v1/content/1?include=owner%2Ctags"
```

Reply:
whoops thanks for the catch